test/corpus/expressions.txt

```diff
@@ -94,3 +94,29 @@ case "12345" {
           (case_clause_pattern
             (discard)))
         (string)))))
+
+================================================================================
+Use
+================================================================================
+
+use a <- f()
```

Comment:
In line with my earlier comment, we should probably have a test for that as well :grin: 

```suggestion
use <- f()

use a <- f()
```